### PR TITLE
PDNS sdk changes

### PR DESCRIPTION
**Pull request by KumarGanesanIBM**

### CODE REVIEW

File: ibm/resource_ibm_private_dns_resource_record.go [hunk 1]
```diff
@@ -71,15 +71,13 @@ func resourceIBMPrivateDNSResourceRecord() *schema.Resource {
...
-				ForceNew:         true,
-				DiffSuppressFunc: caseDiffSuppress,
+				DiffSuppressFunc: suppressPDNSRecordNameDiff,
 				Description:      "DNS record name",
 			},
 
 			pdnsRecordType: {
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
```
Comment (kavya498) [inline ibm/resource_ibm_private_dns_resource_record.go:82]: Name and Record type doesn't seem to have update feature..
Why is force new removed from both attributes?

File: ibm/resource_ibm_private_dns_resource_record.go [hunk 2]
```diff
@@ -517,3 +514,16 @@ func resourceIBMPrivateDNSResourceRecordExists(d *schema.ResourceData, meta inte
 	}
 	return true, nil
 }
+
+func suppressPDNSRecordNameDiff(k, old, new string, d *schema.ResourceData) bool {
+	// CIS concantenates name with domain. So just check name is the same
```
Comment (kavya498) [inline ibm/resource_ibm_private_dns_resource_record.go:519]: Please rename CIS to PDNS

File: ibm/resource_ibm_private_dns_resource_record.go (hunk 1 above)
Comment (KumarGanesanIBM) [reply ibm/resource_ibm_private_dns_resource_record.go]: added ForceNew back

File: ibm/resource_ibm_private_dns_resource_record.go (hunk 2 above)
Comment (KumarGanesanIBM) [reply ibm/resource_ibm_private_dns_resource_record.go]: done

File: ibm/resource_ibm_private_dns_permitted_network.go [hunk 3]
```diff
@@ -121,28 +121,27 @@ func resourceIBMPrivateDnsPermittedNetworkCreate(d *schema.ResourceData, meta in
...
 	response, detail, err := sess.GetPermittedNetwork(getPermittedNetworkOptions)
 
 	if err != nil {
 		return fmt.Errorf("Error reading pdns permitted network:%s\n%s", err, detail)
 	}
 
-	d.Set("id", response.ID)
-	d.Set(pdnsInstanceID, id_set[0])
-	d.Set(pdnsZoneID, id_set[1])
+	d.Set(pdnsPermittedNetworkID, *response.ID)
```
Comment (kavya498) [inline ibm/resource_ibm_private_dns_permitted_network.go:142]: Please remove this..

Comment (KumarGanesanIBM) [reply ibm/resource_ibm_private_dns_permitted_network.go]: removed

**Pull request merged**